deploy/complete/helm-chart/mushop/charts/nats/templates/nats-deployment.yaml

```diff
@@ -22,6 +22,9 @@ spec:
         run: {{ include "nats.name" . }}
       annotations:
         sidecar.istio.io/rewriteAppHTTPProbers: "true"
+        prometheus.io/scrape: "true"
```

Comment:
scrape or exporter?

Reply:
scrape - `prometheus-nats-exporter` simply provides a scrape endpoint. The helm chart version can configure prom using the prom operator, in any case, the mechanism is always scrape.